imperatives/imperatives.go

```diff
@@ -660,89 +671,109 @@ func readModRoute(s *toki.Scanner, table *tbl.Table) error {
 	return table.UpdateRoute(key, opts)
 }
 
+func readDestination(s *toki.Scanner, table Table, allowMatcher bool) (dest *destination.Destination, err error) {
```

Comment:
can you add this new function under readDestinations, that will make it easier to compare old/new. also in general I think  smaller sub-functions should go under the ones where they're being called from.